Makefile.generated_files

```diff
@@ -253,20 +253,18 @@ $(DEEPCOPY_FILES): $(DEEPCOPY_GEN)
 # file if the contents have actually changed.  We 'sinclude' this later.
 .PHONY: $(META_DIR)/$(DEEPCOPY_GEN).mk
 $(META_DIR)/$(DEEPCOPY_GEN).mk:
-	mkdir -p $(@D);                                        \
-	(echo -n "$(DEEPCOPY_GEN): ";                          \
-	 DIRECT=$$(go list -f '{{.Dir}} {{.Dir}}/*.go'         \
-	     ./cmd/libs/go2idl/deepcopy-gen);                  \
-	 INDIRECT=$$(go list                                   \
-	     -f '{{range .Deps}}{{.}}{{"\n"}}{{end}}'          \
-	     ./cmd/libs/go2idl/deepcopy-gen                    \
-	     | grep --color=never "^$(PRJ_SRC_PATH)"           \
-	     | sed 's|^$(PRJ_SRC_PATH)|./|'                    \
-	     | xargs go list -f '{{.Dir}} {{.Dir}}/*.go');     \
-	 echo $$DIRECT $$INDIRECT                              \
-	     | sed 's/ / \\=,/g'                               \
-	     | tr '=,' '\n\t';                                 \
-	) | sed "s|$$(pwd -P)/||" > $@.tmp;                    \
+	mkdir -p $(@D);                                                       \
```

Comment:
I know you hate bash scripts, but there is some real copy/pasta going on here.  Future idea: break this pipeline out in to a small script that can be documented well and reused?